scripts/prepare-doc.sh

```diff
@@ -6,17 +6,23 @@ WORK_DIR=$3
 if [ -z "$PV_SRC" ] || [ -z "$PV_BUILD" ] || [ -z "$WORK_DIR" ]
 then
     echo "Usage:"
-    echo "  ./prepare-doc.sh /path/to/paraview/src /path/to/paraview/build /path/to/workdir"
+    echo "  ./prepare-doc.sh /path/to/paraview/src /path/to/paraview/build /path/to/workdir [version]"
     echo ""
     exit 0
 fi
 
 # -----------------------------------------------------------------------------
 # Source version
 # -----------------------------------------------------------------------------
+if [ -z $4 ]
+then
+    VERSION=$4
+else
+    cd $PV_SRC
+    VERSION=`git describe`
+    cd -
```

Comment:
Actually, this just stays in `$PV_SRC` which seems fine; it used to do that too. The other side *doesn't* change. Is that a change in assumptions?

If not `git -C $PV_SRC describe` may be a better solution.